packages/salesforcedx-vscode-soql/test/vscode-integration/client/completion.test.ts

```diff
@@ -5,18 +5,23 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { expect } from 'chai';
+import { Connection } from 'jsforce';
```

Comment:
Do we need to use the 'Connection' from jsforce specifically? We typically access the 'Connection' object from `@salesforce/core`

Reply:
Changed. Thanks!